common/addons/library-image/src/main/java/com/dfsek/terra/addons/image/config/colorsampler/mutate/RotateColorSamplerTemplate.java

```diff
@@ -0,0 +1,19 @@
+package com.dfsek.terra.addons.image.config.colorsampler.mutate;
+
+import com.dfsek.tectonic.api.config.template.annotations.Value;
+import net.jafama.FastMath;
+
+import com.dfsek.terra.addons.image.colorsampler.ColorSampler;
+import com.dfsek.terra.addons.image.colorsampler.mutate.RotateColorSampler;
+
+
+public class RotateColorSamplerTemplate extends MutateColorSamplerTemplate {
+    
+    @Value("angle")
+    private double angle;
+    
+    @Override
+    public ColorSampler get() {
+        return new RotateColorSampler(sampler, FastMath.toRadians(angle));
```

Comment:
Perhaps add special-case optimisations for 90, 180 and 270 degree rotations, as these will probably be very common, and trig is expensive.

Reply:
Implemented